src/dsp.jl

```diff
@@ -410,3 +411,64 @@ function mfilter(r, s)
   n = length(r) - 1
   sfilt(f, padded(s, (0, n)))[n+1:end]
 end
+
+"""
+$(SIGNATURES)
+Inverse Short Time Fourier Transform (ISTFT). 
+
+The signal windowing must obey the constraint of "nonzero overlap add" (NOLA). 
+Implementation based on Hristo (2019, 2020) and `istft` in `scipy`. 
+
+H. Zhivomirov. On the Development of STFT-analysis and ISTFT-synthesis Routines 
+and their Practical Implementation. TEM Journal, ISSN: 2217-8309, DOI: 10.18421/TEM81-07, 
+Vol. 8, No. 1, pp. 56-64, Feb. 2019. 
+(http://www.temjournal.com/content/81/TEMJournalFebruary2019_56_64.pdf)
+
+Hristo Zhivomirov (2020). Inverse Short-Time Fourier Transform (ISTFT) with Matlab 
+(https://www.mathworks.com/matlabcentral/fileexchange/45577-inverse-short-time-fourier-transform-istft-with-matlab), 
+MATLAB Central File Exchange. Retrieved October 21, 2020. 
+"""
+function istft(X::AbstractMatrix{Complex{T}}, 
```

Comment:
Would be good to have more documentation on parameters and perhaps some examples. It isn't obvious to me how it's meant to be used. Also is `window` compatible with all other uses of window functions in `SignalAnalysis`? Also consider making the non-required params as keyword params, and keep the convention same as other functions that use `noverlap`, `window`, etc.

Reply:
The docstring has been improved by adding more descriptions and an example. `window` is compatible with all other uses of `DSP.Periodograms`. `n` and `noverlap` are positional arguments which are based on the convention same as `DSP.Periodograms.stft`.

Reply:
See https://github.com/org-arl/SignalAnalysis.jl/blob/94b235a85b9c5be16a84c4782cea00a2d54c55aa/src/basic.jl#L42 for style for `window` and `nfft` that is currently used in other parts of this package. While consistency with `DSP.jl` would be nice to have, it's even more important to have internal consistency.

Reply:
Agreed. Renamed `n` to `nfft`, and set `nfft` and `noverlap` to keyword arguments.